api/Document.json

```diff
@@ -6481,7 +6481,7 @@
             "deprecated": false
           }
         },
-        "deep_optional": {
+        "deep_parameter_optional": {
```

Comment:
I'll revert this change and send a separate PR. Since the Firefox versions are set this smells a lot like a "we changed something and should document it" entry that stops being useful very soon after the Firefox release.